src/com/esotericsoftware/kryo/serializers/TaggedFieldSerializer.java

```diff
@@ -48,7 +51,24 @@
 	private boolean[] deprecated;
 
 	public TaggedFieldSerializer (Kryo kryo, Class type) {
-		super(kryo, type);
+		super(kryo, type, null, kryo.getFieldSerializerConfig());
+	}
+
+	/** Tells Kryo, if should ignore unknown field tags when using TaggedFieldSerializer. Already existing serializer instances
+	 * are not affected by this setting.
+	 *
+	 * <p>
+	 * By default, Kryo will throw KryoException if encounters unknown field tags.
+	 * </p>
+	 *
+	 * @param ignoreUnknownTags if true, unknown field tags will be ignored. Otherwise KryoException will be thrown */
+	public void setIgnoreUnknownTags (boolean ignoreUnknownTags) {
+		((TaggedFieldSerializerConfig) config).setIgnoreUnknownTags(ignoreUnknownTags);
+		rebuildCachedFields();
+	}
+
+	public boolean getIgnoreUnkownTags() {

```

Comment:
This might also be `isIgnoreUnkownTags`. In `FieldSerializer` I didn't change boolean getters to "is" just to not break binary compatibility. For new boolean properties I'd prefer the "is" prefix instead of "get".